### PR TITLE
OSOE-84: Migrate all projects to NE ⭐

### PR DESCRIPTION
**Pull request by 0liver**
OSOE-84

Fixes https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/48.

### CODE REVIEW

File: test/Lombiq.OSOCE.Tests.UI/Lombiq.OSOCE.Tests.UI.csproj [hunk 1]
```diff
@@ -8,6 +8,10 @@
       <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Compile Remove="Tests\BehaviorVueTests.cs" />
```
Comment (sarahelsaig) [inline test/Lombiq.OSOCE.Tests.UI/Lombiq.OSOCE.Tests.UI.csproj]: What is this?

Comment (0liver) [reply test/Lombiq.OSOCE.Tests.UI/Lombiq.OSOCE.Tests.UI.csproj]: 🧾 Once upon a time, there were UI tests that referenced projects which were excluded from the solution... 🧾

~Fixed.~

I'm not exactly sure why this test still fails, but I expect OSOE-92 to have fixed it, so I won't waste time on doing the same. Objections?

Comment (sarahelsaig) [reply test/Lombiq.OSOCE.Tests.UI/Lombiq.OSOCE.Tests.UI.csproj]: Then let's return to this when [OSOE-92](https://lombiq.atlassian.net/browse/OSOE-92) is merged

Comment (0liver) [reply test/Lombiq.OSOCE.Tests.UI/Lombiq.OSOCE.Tests.UI.csproj]: I actually had forgotten to add the VueJs.Samples project to the Web project - that's why the UI test couldn't open the URL from there! That's fixed now.

**Review by sarahelsaig — APPROVED** [review]

**Pull request merged**